services/app-api/src/emailer/emails/unlockPackageStateEmail.ts

```diff
@@ -19,9 +19,10 @@ export const unlockPackageStateEmail = async (
     submitterEmails: string[]
 ): Promise<EmailData | Error> => {
     const isTestEnvironment = config.stage !== 'prod'
-    const stateContactEmails = formData.stateContacts.map(
-        (contact) => contact.email
-    )
+    const stateContactEmails: string[] = []
+    formData.stateContacts.forEach((contact) => {
```

Comment:
If you wanted to keep using map semantics you could add a filter here instead. But this totally works.